### PR TITLE
add hypershift-mce-agent-info

### PR DESCRIPTION
**Pull request by LiangquanLi930**
add step `hypershift-mce-agent-info` to better debugging, log: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/48452/rehearse-48452-periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-mce-agent-connected-dualstack-manual-conformance/1754728932055191552/artifacts/e2e-mce-agent-connected-dualstack-manual-conformance/hypershift-mce-agent-info/build-log.txt

### CODE REVIEW

File: ci-operator/step-registry/hypershift/mce/agent/info/OWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
+approvers:
```
Comment (jparrill) [inline ci-operator/step-registry/hypershift/mce/agent/info/OWNERS:1]: I think @CrystalChun and @carbonin should be part of the reviewers/approvers. WDYT?

File: ci-operator/step-registry/hypershift/mce/agent/info/hypershift-mce-agent-info-ref.metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+	"path": "hypershift/mce/agent/info/hypershift-mce-agent-info-ref.yaml",
+	"owners": {
+		"approvers": [
+			"csrwng",
```
Comment (jparrill) [inline ci-operator/step-registry/hypershift/mce/agent/info/hypershift-mce-agent-info-ref.metadata.json:5]: I think @CrystalChun and @carbonin should be part of the reviewers/approvers?

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-mce-agent-connected-dualstack-manual-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-mce-agent-connected-dualstack-manual-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-mce-agent-connected-dualstack-manual-conformance

**Pull request merged**